Content.Server/Silicons/Laws/SiliconLawSystem.cs

```diff
@@ -154,6 +154,8 @@ private void OnIonStormLaws(EntityUid uid, SiliconLawProviderComponent component
 
     private void OnEmagLawsAdded(EntityUid uid, SiliconLawProviderComponent component, ref GotEmaggedEvent args)
     {
+        if (args.Handled)
+            return;
```

Comment:
Since the other subscription below marks it as handled, wouldn't it mean that this one is skipped sometimes if it happens afterwards? Both are important, but I'm not sure why we need two different ones in the first place. They can likely be combined.

Reply:
Because one on below is an override of SharedSiliconLawSystem's method.